Makefile

```diff
@@ -37,7 +37,7 @@ ifeq ($(OS),Windows_NT)
 else
 	UNAME_OS := $(shell uname -s)
 	UNAME_ARCH := $(shell uname -m)
-	PROTOC_BUILD := $(shell echo ${UNAME_OS}-${UNAME_ARCH} | tr '[:upper:]' '[:lower:]' | sed 's/darwin/osx/' | sed 's/aarch64/aarch_64/')
+	PROTOC_BUILD := $(shell echo ${UNAME_OS}-${UNAME_ARCH} | tr '[:upper:]' '[:lower:]' | sed 's/darwin/osx/' | sed 's/arm64/aarch_64/')
```

Comment:
You need to leave the `| sed 's/aarch64/aarch_64/'` or it will stop working on arm64 linux as:
```fish
$ uname -s
Linux
$ uname -m
aarch64
$ uname -a
Linux arm64-runner-uscentral1 5.15.0-1022-gcp #29-Ubuntu SMP Mon Oct 24 21:00:05 UTC 2022 aarch64 aarch64 aarch64 GNU/Linux
```